src/eip4824.ts

```diff
@@ -0,0 +1,157 @@
+import express from 'express';
+import { getSpace } from './helpers/spaces';
+import db, { sequencerDB } from './helpers/mysql';
+
+const router = express.Router();
+
+router.get('/:space', async (req, res) => {
+  let space: any = {};
+  const protocol = req.get('X-Forwarded-Proto') || req.protocol;
+  const host = req.get('X-Forwarded-Host') || req.get('host');
+  const baseUrl = `${protocol}://${host}${req.originalUrl}`;
+
+  try {
+    space = await getSpace(req.params.space);
+
+    if (!space.verified) return res.status(404).json({ error: 'INVALID' });
+  } catch (e) {
+    console.log(e);
+    return res.status(500).json({ error: 'NOT_FOUND' });
+  }
+
+  return res.json({
+    '@context': 'http://www.daostar.org/schemas',
```

Comment:
it is like this `<http://www.daostar.org/schemas>` 
Refer: 
- https://cloudflare-ipfs.com/ipfs/QmWCqZZDEQQqrEctrEDRpqjtLDKhrPpGgereGLfXuMPnJ1
- https://api.daostar.org/immutable/Qmd8N6ShPgrhyExDNimt4ycWnyw93wqVAa1cuS7GnCpWyq

Also, i feel like better to have a constant and re-use everywhere